worker/src/RTC/RateCalculator.cpp

```diff
@@ -132,6 +132,11 @@ namespace RTC
 		uint64_t nowMs = DepLibUV::GetTimeMs();
 
 		this->packets++;
-		this->rate.Update(packet->GetSize(), nowMs);
+
+		// Ignore the bitrate of the padding only packets as they won't be forwarded.
+		if (packet->GetPayloadLength() > 0)
+		{
+			this->rate.Update(packet->GetSize() - packet->GetPayloadPadding(), nowMs);
```

Comment:
So here you are subtracting the payload padding, which makes sense. But AFAIR ((I may be wrong) we do not remove the padding when forwarding the RTP packet to consuming endpoint. Do we?

Reply:
Correct, this PR doesn't remove the padding so the support for packets with payload + padding is not complete in this PR but those packets are not used by libwebrtc and didn't know how/where to make that manipulation of packets.

So this PR only implements properly (in theory) the support for packets that are all padding (no payload) that are the ones used by libwebrtc right now afaict.

Reply:
Is this strictly necessary for this PR @ggarbe?

Reply:
no, it can be done in a separate PR if needed